SystemInformer/searchbox.c

```diff
@@ -1211,6 +1215,17 @@ LRESULT CALLBACK PhpSearchWndSubclassProc(
                     }
                 }
             }
+            if (wParam == VK_ESCAPE)
+            {
+                PhSetWindowText(hWnd, L"");
+                PhpSearchUpdateText(hWnd, context, FALSE);
+                if (context->PreviousFocusWindowHandle)
+                {
+                    SetFocus(context->PreviousFocusWindowHandle);
+                    context->PreviousFocusWindowHandle = NULL;
+                }
+                return 1;
+            }
```

Comment:
Would it be possible to make it so up and down arrows also return focus to the previous window (without clearing the text)? Those are common keys to navigate up and down in the tree view but are not generally keys used for navigating the single row search box. That also might provide a nicer user experience to break away from the search box.

Reply:
Good idea, done. 
Seem to work nicely.